tests/unit/__init__.py

```diff
@@ -11,3 +11,30 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+import copy
+from tests import unittest
+
+
+class BaseResponseTest(unittest.TestCase):
+    def assert_response(self, actual_response, expected_response):
```

Comment:
Can we rename this to something more explicit it. ``assert_response`` seemed vague to me when I read it. I think my main issue was that I did not realize you do a subset check to assert the response. I would have thought it would be an equality check. Maybe something like ``assert_response_with_subset_metadata``?